src/wp-includes/functions.php

```diff
@@ -8409,3 +8409,111 @@ function is_php_version_compatible( $required ) {
 function wp_fuzzy_number_match( $expected, $actual, $precision = 1 ) {
 	return abs( (float) $expected - (float) $actual ) <= $precision;
 }
+
+/**
+ **
+ * The number of active users in your installation.
+ *
+ * The count is cached and updated twice daily. This is not a live count.
+ *
+ * @since MU (3.0.0)
+ * @since 4.8.0 The `$network_id` parameter has been added.
+ * @since 6.0.0 Make to function.php.
```

Comment:
Minor typo and accuracy improvement:
```suggestion
 * @since 6.0.0 Move to wp-includes/functions.php.
```